src/TyTTP/Core/Context.idr

```diff
@@ -18,7 +18,11 @@ export
 Bifunctor (Context me u v h1 s h2) where
   bimap f g step = { request $= map f, response $= map g } step
 
-infixr 0 :>, :>>
+private
```

Comment:
If I use this change `:>` cause errors in tests.

eg: [json/tests/json/server/JSONServer.idr](https://github.com/kbertalan/tyttp/blob/ede05cbef5c662132f201f9a9408c82c0673a16f/json/tests/json/server/JSONServer.idr#L30)

```
1/1: Building JSONServer (JSONServer.idr)
Error: Unknown operator ':>'

JSONServer:30:5--30:7
 26 |   http <- HTTP.require
 27 |   server <- HTTP.listen'
 28 |     $ (\next, ctx => mapFailure message (next ctx))
 29 |     $ parseUrl' (const $ sendText "URL has invalid format" >=> status BAD_REQUEST)
 30 |     :> routes' (sendText "Resource could not be found" >=> status NOT_FOUND) { m = Promise Error IO }
          ^^
```

Reply:
Ah I think I know what's wrong, those operators need `export`, not `private` (I'm not by my computer right now, but will fix this ASAP).

Reply:
@kbertalan 

I think this should build now.